### ISSUE TITLE
Accessibility Request - Move files in File Explorer Sidebar for keyboard users

### ISSUE THREAD

**Issue by garrettbsettles** [microsoft/vscode#124775]
**Problem**:
Currently to move files in and out of folders in side bar file explorer, a user must use a mouse to drag and drop files into desired folder. This poses an accessibility issue to keyboard users - If a user cannot use a mouse, a user cannot drag and drop files.

**Solution**:
When File Explorer Side Bar is focused, move files up or down with keyboard shortcuts option up or down or with Command Palette "Move File Up" "Move File Down". If Move file up is selected, it will move the file first to the next closest shown folder. If the folder is focused and collapsed, it will be expanded. However, folders that were expanded will collapse once they are no longer focused. As long as the option key is depressed, the file selected for the move will move into each folder directly underneath the containing folder name when the up or down arrow keys are depressed, ignore file name sort order until option key is no longer pressed, at which point the view will focus on the file that has been moved and its position with the folder's file list will be correctly sorted.

**Notes**
It would be easier to implement "Move File Next (Sibling) Folder", "Move File Previous (Sibling) Folder", "Move File Up Folder (Level)", "Move File Down Folder (Level)", however users are used to using option up or down to move lines in the code editor, and expecting the same / similar behavior in the file explorer would increase user adoption. Or perhaps an alternative solution could be-

"Move File Next (Sibling) Folder" - option down
"Move File Previous (Sibling) Folder" - option up
"Move File Down Folder (Level)" - option right
"Move File Up Folder (Level)" - option left

Something is better than the current status quo though which leaves keyboard users out

**Comment by isidorn** [comment]
@garrettbsettles thanks for your feedback.

If you want to move files around the explorer why is the following scenario not working for you: press `Ctrl + X` to cut files, then use the keyboard to move to the destination and press `Ctrl + V` to paste them?

**Comment by garrettbsettles** [comment]
@isidorn ah - your recommendation is exactly what I was looking for - thank you. 

Also, I tried looking for documentation online for moving files around via the keyboard but couldn't find anything. Could you post a link to the documentation where it mentions moving files around via the keyboard? I think it could help with future users who run into a similar issue.

**Comment by isidorn** [comment]
@garrettbsettles we do not currently mention it in our docs between that keybinding is pretty common when using file explorer.
However if you are passionate about this we accept PRs that improve our docs https://github.com/microsoft/vscode-docs

I will close this for now since no code changes are planned here
